g11n-ws/modules/md-service-i18n-l2/src/main/java/com/vmware/i18n/l2/service/pattern/PatternServiceImpl.java

```diff
@@ -74,6 +75,13 @@
 		}
 		logger.info("get pattern data from cache");
 		patternMap = JSONUtils.getMapFromJson(patternJson);
+		if (StringUtils.isEmpty(patternMap.get(ConstantsKeys.REGION))) {
+			String regionJson = PatternUtil.getRegionFromLib(locale.replace("_", "-"));
+			if (!StringUtils.isEmpty(regionJson)) {
```

Comment:
It will be better to use isNotEmpty method.

Reply:
fixed